test/contractBridge/depositGeneric.js

```diff
@@ -8,37 +8,51 @@ const Ethers = require('ethers');
 
 const RelayerContract = artifacts.require("Relayer");
 const BridgeContract = artifacts.require("Bridge");
+const CentrifugeAssetContract = artifacts.require("CentrifugeAsset");
 const GenericHandlerContract = artifacts.require("GenericHandler");
 
 contract('Bridge - [deposit - Generic]', async (accounts) => {
+    const relayerThreshold = 2;
     const originChainID = 1;
     const destinationChainID = 2;
     const recipientAddress = accounts[1];
     const expectedDepositNonce = 1;
-    const genericBytes = '0x736f796c656e745f677265656e5f69735f70656f706c65';
-
-    const randomAddress = Ethers.utils.hexZeroPad('0x1', 20)
+    const blankFunctionSig = '0x00000000';
+    const centrifugeAssetFuncSig = Ethers.utils.keccak256(Ethers.utils.hexlify(Ethers.utils.toUtf8Bytes('store(bytes32)'))).substr(0, 10);
```

Comment:
This would be a good helper function, these could also live inside constants as opposed to being redefined

Reply:
I'll make sure to move these when doing #91